examples/fluid/static_sphere_shape.jl

```diff
@@ -0,0 +1,92 @@
+# In this example we try to approach the static shape of a water droplet on a horizontal plane.
+# The shape of a static droplet can be calculated from the Young-Laplace equation.
+using TrixiParticles
```

Comment:
1. I don't like the name. It's not a static example, so that's misleading. It's just as static as the `sphere_surface_tension`. Maybe `sphere_surface_tension_wall_2d.jl` or so? Because it's essentially the same as sphere surface tension, just on the ground, right?
2. Can't you just `trixi_include` the falling spheres example here? This would also make it clearer what is different in this example. Like:
```julia
using TrixiParticles
using OrdinaryDiffEq

fluid_particle_spacing = 0.0025
tspan = (0.0, 0.3)
tank_size = (1.0, 0.1)
sound_speed = 120.0
nu = 0.00025
sphere_radius = 0.05
sphere1_center = (0.5, sphere_radius)

trixi_include(@__MODULE__,
              joinpath(examples_dir(), "fluid", "falling_water_spheres_2d.jl"),
              sphere=nothing, ...,)
```